### PR TITLE
[ZEPPELIN-447] Document review process and becoming a committer

### PR DESCRIPTION
**Pull request by Leemoonsoo**
https://issues.apache.org/jira/browse/ZEPPELIN-477

Documenting review process and becoming a committer.

Note that purpose of this PR is not improving those process but document what it looks like at the moment.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,8 +11,30 @@ Contributing to Zeppelin (Source code, Documents, Image, Website) means you agre
...
 Zeppelin follows [Fork & Pull] (https://github.com/sevntu-checkstyle/sevntu.checkstyle/wiki/Development-workflow-with-Git:-Fork,-Branching,-Commits,-and-Pull-Request) model.
 
+
+## The Review Process
+
+When a Pull Request is submitted, it is being merged or rejected by following review process.
+
+* Anybody can be a reviewer and may comment on the change and suggest modifications.
+* Reviewer can indicate that a patch looks suitable for merging with a comment such as : "Looks good", "LGTM", "+1".
+* Pullrequest is open for 1 or 2 days for potential additional review, unless it's got enough indication of suitable for merging.
```
Comment (corneadoug) [inline CONTRIBUTING.md]: I think this step should be below, since its after a committer gave an indication of suitable for merging

**Comment by Leemoonsoo** [comment]
If it does reflect current process, please comment.
I'm merging it if there're no more discussions.

**Comment by corneadoug** [comment]
Yes it's good, you can Merge

**Comment by bzz** [comment]
This looks like very solid foundation! :+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,8 +11,30 @@ Contributing to Zeppelin (Source code, Documents, Image, Website) means you agre
...
+
+## The Review Process
+
+When a Pull Request is submitted, it is being merged or rejected by following review process.
+
+* Anybody can be a reviewer and may comment on the change and suggest modifications.
+* Reviewer can indicate that a patch looks suitable for merging with a comment such as : "Looks good", "LGTM", "+1".
+* At least one indication of suitable for merging(eg. "LGTM") from committer is required to be merged.
+* Pullrequest is open for 1 or 2 days for potential additional review, unless it's got enough indication of suitable for merging.
+* Committer can initiate lazy consensus ("Merge if there is no more discusssion") and the code can be merged after certain time (normally 24 hours) when there is no review exists.
```
Comment (prabhjyotsingh) [inline CONTRIBUTING.md]: discusssion - discussion

**Comment by bzz** [comment]
Typo in PR title - it's [ZEPPELIN-477](https://issues.apache.org/jira/browse/ZEPPELIN-477) in JIRA

**Pull request closed without merging**